Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/playwright:v1.45.1-jammy
+
+COPY . .
+
+RUN npm install
+RUN npx playwright install
```

Comment:
do we have to install playwright like this or can we not have it in the package.json file instead and have `npm install` command pick it up form there?

Reply:
yep you're right, just tested it without and it installed fine so I'll remove that from there and the makefile

Reply:
made changes in upcoming commit